src/api/routes/offer.js

```diff
@@ -58,6 +59,19 @@ module.exports = (app) => {
         }
     });
 
+    /**
+     * Gets all the offers of a certain company from the db
+     */
+    router.get("/company/:companyId", companyValidators.validCompanyId, async (req, res, next) => {
```

Comment:
I'm wondering if we should verify if the company really exists and throw an error, instead of returning an empty array

Reply:
I agree, even though the current behavior is not entirely wrong (An inexistent company technically has no offers :P)